lumicks/pylake/file.py

```diff
@@ -209,3 +210,34 @@ def fdcurves(self) -> Dict[str, FDCurve]:
     @property
     def markers(self) -> Dict[str, Marker]:
         return self._get_object_dictionary("Marker", Marker)
+
+    def export_h5(self, filename, compression_level=5, omit_data={}):
+        """Write a modified h5 file to disk.
+
+        When transferring data, it can be beneficial to omit some channels from the h5 file, or use a higher compression
+        ratio. In particular, high frequency channels tend to take up a lot of space, and aren't always necessary for
+        every single analysis.
```

Comment:
It would be nice to mention here that Bluelake exports files at compression level 1 for performance reasons so this function can help reduce the size further if needed.

Reply:
👍